src/realm/sync/noinst/reopening_file_logger.cpp

```diff
@@ -18,7 +18,7 @@ ReopeningFileLogger::ReopeningFileLogger(std::string path, volatile std::sig_ato
 }
 
 
-void ReopeningFileLogger::do_log(util::Logger::Level level, std::string message)
+void ReopeningFileLogger::do_log(util::Logger::Level level, std::string const& message)
```

Comment:
We go for `const std::string&` everywhere else (except for some reason not in Object Store)